content/departments/product-engineering/engineering/cloud/delivery/managed/enable_executors_process.md

```diff
@@ -0,0 +1,87 @@
+# Enabling executors
+
+Enabling executor on a managed instances is initiated by "?". There are a few logical steps to follow.
+
+1. Update the managed instances terraform to include executor modules
+1. Setup billing alert for the executor resources
+
+## Update the managed instances terraform
+
+The change to terraform will be done by Batch Change team. Batch Change teamm will ping us the Pull Request for the changes and Delivery team will apply the change.
```

Comment:
```suggestion
The change to terraform will be done by Batch Change team. Batch Change team will ping Delivery on the Pull Request for the changes and Delivery team will apply the change.
```

Reply:
https://github.com/sourcegraph/deploy-sourcegraph-managed/pull/311#issuecomment-1049794731 - the process isn't quite as straightforward as 'terraform apply' yet, might be worth linking this until we can improve things